src/fa.h

```diff
@@ -53,6 +53,19 @@ enum fa_minimization_algorithms {
  */
 extern int fa_minimization_algorithm;
 
+/* Data structure defining the exported automata as a list*/
+struct FA_EXPORT {
+    int state; // Current state
```

Comment:
Removing state number implies modifying the edges so for each transition we store a pointer to the destiny rather than an identifier.
Using these numbers reduces the memory usage and, in my opinion, makes the structure easier to manipulate.

Anyways, I will change the variable type.